recipe/meta.yaml

```diff
@@ -34,11 +34,12 @@ about:
   home: https://libexpat.github.io
   license: MIT
   license_family: MIT
-  # Note that the file is in a subdirectory expat/COPYING the expat is in a subdirectory
   license_file: COPYING
-  summary: 'Expat XML parser library in C'
+  license_url: https://github.com/libexpat/libexpat/blob/master/expat/COPYING
```

Comment:
For v2.4.9, it was copied to libexpat/COPYING, see https://github.com/libexpat/libexpat/commit/34f57a0b3033bd6eff689e613ebf3eed56b2d1c7

Reply:
They placed it at two paths. 
https://github.com/libexpat/libexpat/blob/R_2_4_9/expat/COPYING
https://github.com/libexpat/libexpat/blob/R_2_4_9/COPYING